packages/sitecore-jss-react/src/components/ErrorBoundary.tsx

```diff
@@ -33,19 +33,32 @@ class ErrorBoundary extends React.Component<ErrorBoundaryProps> {
   }
 
   componentDidCatch(error: Error, errorInfo: React.ErrorInfo) {
+    if (this.showErrorDetails()) {
+      console.error(
+        `A rendering error occurred in component ${this.props.rendering?.componentName}; Rendering uid: ${this.props.rendering?.uid}`
```

Comment:
Let's make it similar to existing error: https://github.com/Sitecore/jss/blob/dev/packages/sitecore-jss-nextjs/src/services/component-props-service.ts#L185
```suggestion
        `A rendering error occurred in component ${this.props.rendering?.componentName} (${this.props.rendering?.uid})`
```